### PR TITLE
ENH: Allow afni.MaskTool to take multiple input files

### LINKED ISSUE #2874: AFNI's 3dmask_tool does not accept multiple mask files
### Summary

As far as I can see AFNI's `3dmask_tool` allows multiple masks as input, while it is not possible to input a list as `in_file` argument?

See [here](https://afni.nimh.nih.gov/pub/dist/doc/program_help/3dmask_tool.html) for details on the tool.

Error:
```python
Traceback (most recent call last):
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/nipype/pipeline/plugins/linear.py", line 44, in run
    node.run(updatehash=updatehash)
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/nipype/pipeline/engine/nodes.py", line 408, in run
    cached, updated = self.is_cached()
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/nipype/pipeline/engine/nodes.py", line 294, in is_cached
    hashed_inputs, hashvalue = self._get_hashval()
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/nipype/pipeline/engine/nodes.py", line 488, in _get_hashval
    self._get_inputs()
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/nipype/pipeline/engine/nodes.py", line 531, in _get_inputs
    self.set_input(key, deepcopy(output_value))
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/nipype/pipeline/engine/nodes.py", line 276, in set_input
    setattr(self.inputs, parameter, deepcopy(val))
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/nipype/interfaces/base/traits_extension.py", line 103, in validate
    validated_value = super(File, self).validate(object, name, value)
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/traits/trait_types.py", line 411, in validate
    self.error( object, name, value )
  File "/home/hstojic/.pyenv/nipy/local/lib/python2.7/site-packages/traits/trait_handlers.py", line 172, in error
    value )
TraitError: The 'in_file' trait of a MaskToolInputSpec instance must be an existing file name, but a value of ['mask_1',...,'mask_n'] <type 'list'> was specified.
```

### Execution environment

- My python environment outside container

### PR DESCRIPTION
**Pull request by hstojic**
## Summary

Fixes #2874. 

## List of changes proposed in this PR (pull-request)

Made a change to the inputs of `MaskTool` class in AFNI interface. Used `InputMultiObject` as suggested, tested it out and now it works fine with multiple input images as well as far as I can tell.

## Acknowledgment

- [x] \(Mandatory\) I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

File: nipype/interfaces/afni/utils.py [hunk 1]
```diff
@@ -1636,12 +1636,11 @@ def _format_arg(self, name, spec, value):
 
 
 class MaskToolInputSpec(AFNICommandInputSpec):
-    in_file = File(
-        desc='input file or files to 3dmask_tool',
+    in_file = InputMultiPath(
+        File(desc='input file or files to 3dmask_tool', exists=True),
```
Comment (effigies) [inline nipype/interfaces/afni/utils.py]: `desc` should be an argument to `InputMultiPath`, not `File`, in this case.

File: nipype/algorithms/modelgen.py [hunk 2]
```diff
@@ -414,6 +418,12 @@ def _generate_standard_design(self,
                     if hasattr(info, 'tmod') and info.tmod and \
                             len(info.tmod) > cid:
                         sessinfo[i]['cond'][cid]['tmod'] = info.tmod[cid]
+                    
+                    if hasattr(info, 'orth') and info.orth:
+                        if info.orth[cid] == 'Yes':
+                            sessinfo[i]['cond'][cid]['orth'] = 1
+                        elif info.orth[cid] == 'No':
+                            sessinfo[i]['cond'][cid]['orth'] = 0
```
Comment (effigies) [inline nipype/algorithms/modelgen.py]: Can you rebase or revert the `orth` changes out?

Comment (hstojic) [reply nipype/algorithms/modelgen.py]: ah, silly, I checked out a new branch from another patch :|

Comment (hstojic) [reply nipype/algorithms/modelgen.py]: oops, I did a hard reset to SHA before the `orth` changes and then force-push to update the remote branch, but I didnt realize that one will close the pull request. I committed the AFNI changes again, but they dont appear here anymore.

Shall I simply do another pull request?

**Comment by effigies** [comment]
Feel free to open a new PR, if that's easiest.

File: nipype/interfaces/afni/utils.py (hunk 1 above)
Comment (effigies) [reply nipype/interfaces/afni/utils.py]: Yeah, not everything is consistent or correct.

**Comment by hstojic** [comment]
Ok, opened a new PR, at #2892

**Pull request closed without merging**